lib/api.js

```diff
@@ -87,6 +87,21 @@ PlexAPI.prototype.postQuery = function query(options) {
     return this._request(options).then(uri.attach(url));
 };
 
+PlexAPI.prototype.putQuery = function query(options) {
```

Comment:
..see we've got the same mistake in the `postQuery()` function above. Could you please fix that one as well?